pkg/arvo/sys/zuse.hoon

```diff
@@ -5408,13 +5417,20 @@
     ::                                                  ::::
   ++  chrono  ^?
     |%
-    ::  +from-unix: unix timestamp to @da
+    ::  +from-unix-s: unix seconds to @da
     ::
-    ++  from-unix
+    ++  from-unix-s
```

Comment:
idk if this renaming is worth it, or even necessary. "unix timestamp" generally means second resolution. At the very least, consider adding `++  from-unix  from-unix-s` to preserve backwards compatibility.

Reply:
I like the addition of a distinction between `+from-unix` and `+from-unix-ms`. I don't see a need for `+from-unix-s`, though.

Reply:
Yeah, I wondered about that. I'll fix it.